sdk/keyvault/azure-keyvault-keys/tests/test_samples_keys_async.py

```diff
@@ -47,7 +54,7 @@ async def test_example_key_crud_operations(self, client, **kwargs):
 
         # create a key with optional arguments
         key = await key_client.create_key(
-            "key-name", "RSA", size=key_size, key_operations=key_ops, expires_on=expires_on
+            key_name, "RSA", size=key_size, key_operations=key_ops, expires_on=expires_on
```

Comment:
Don't recall why we ended up with a string here. It seems better to demonstrate the enum in snippets:
```suggestion
            key_name, KeyType.rsa, size=key_size, key_operations=key_ops, expires_on=expires_on
```